### PR TITLE
Normalized tagging, removed keyboard item

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-2013#bug-fixes

Changes made by this PR:
* Added statement about platforms the page applies to.
* Trimmed trailing spaces.
* Put divider comment on line above h2.
* Sentence-cased headings.

before:
https://docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes#bug-fixes

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -35,18 +35,23 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
 
+<!-- ====================================================================== -->
+## Platforms and languages covered
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:39]: @peiche-jessica - pls review this section

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -62,24 +67,22 @@ The following items are now stable:
 
 *  The [HttpStatusCode API](/microsoft-edge/webview2/reference/win32/icorewebview2navigationcompletedeventargs2?view=webview2-1.0.1245.22&preserve-view=true) which provides the HTTP status code for navigation requests in `NavigationCompleted` events.
 
-
 #### Bug Fixes
-  
-*   Fixed an issue with the on-screen keyboard in which the keyboard does not reappear once it is closed by clicking the **X** button. Also fixed an issue in which the keyboard gets dismissed when users switch from one edit control to another within WebView2. ([Issue #460](https://github.com/MicrosoftEdge/WebView2Feedback/issues/460))
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:68]: @nishitha-burman - this is the removed item

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -35,18 +35,23 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
 
+<!-- ====================================================================== -->
+## Platforms and languages covered
+
+Generally, release notes apply to both Win32/C++ and .NET/C#.  In some cases, a release note specifies a particular platform that it applies to.  The [Win32/C++ API Reference](/microsoft-edge/webview2/reference/win32/icorewebview2) is parallel with the [.NET/C# API Reference](/dotnet/api/microsoft.web.webview2.core.corewebview2).
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:41]: IIUC release notes can apply to all Win32, .NET, and WinRT. Win32 `ICoreWebView2` and `ICoreWebView2_N` interfaces are roughly parallel to .NET and/or WinRT CoreWebView2 class.

**Review by nishitha-burman — APPROVED** [review]

**Review by vchapel — APPROVED** [review]
LGTM

**Review by peiche-jessica — APPROVED** [review]
LGTM.

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -62,24 +70,23 @@ The following items are now stable:
...
-*   Fixed regression where WebView2 would steal focus from the app when the WebView2 was made visible.  ([Issue #862](https://github.com/MicrosoftEdge/WebView2Feedback/issues/862)) 
-*   Fixed a bug that caused increased memory usage with `WebResourceRequested` events using large data. ([Issue #2171](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2171)) 
-*   Fixed `StatusBarTextChanged` regression. The [StatusBarText API](/microsoft-edge/webview2/reference/win32/icorewebview2_12?view=webview2-1.0.1245.22&preserve-view=true) was made compatible with previous versions again. ([Issue #2414](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2414))   
-*   Better support for apps running as admin. ([Issue #2356](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2356)) 
+*  Fixed an issue with the on-screen keyboard in which the keyboard doesn't reappear after it's closed by clicking the **X** button. Also fixed an issue in which the keyboard gets dismissed when users switch from one edit control to another within WebView2. ([Issue #460](https://github.com/MicrosoftEdge/WebView2Feedback/issues/460))
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:75]: @nishitha-burman - pls review this list item.  whitespace changes, minor copedit.

**Pull request merged**